doc-content/enterprise-only/integration/dev-studio-install-proc.adoc

```diff
@@ -3,6 +3,11 @@
 
 Red Hat Developer Studio is an integrated development environment (IDE) based on Eclipse. It integrates tooling and run time components by combining Eclipse, Eclipse Tooling, and {EAP_LONG}. Red Hat Developer Studio provides plug-ins with tools and interfaces for {PRODUCT}.
 
+[WARNING]
+====
+Red Hat Developer Studio is in maintenance mode. Installing and setting up the development environment with Red Hat JBoss Developer Studio for {PRODUCT} {PRODUCT_VERSION} is deprecated.
+====
```

Comment:
@csherrar It took me a couple of reads to realize that the second sentence refers to the previous version "Red Hat JBoss Developer Studio", while the first refers to the new version "Red Hat Developer Studio". Isn't that what you're saying? Maybe we could make this clearer by saying "Red Hat JBoss Developer Studio (legacy)" or include version numbers, if applicable, or something. Otherwise it reads at first like "Dev Studio is in maintenance mode. Dev Studio is actually deprecated." (Wait, what?).

Reply:
This is my mistake. The notice refers to the same version, I accidentally included "JBoss" in the second sentence. The first sentence refers to Developer Studio itself, and the second refers to the installing and setting up a development environment with Developer Studio/. In this case, I can just remove "JBoss" but that may not make it any clearer. What are your thoughts?

Reply:
I guess the note is okay then, but I suggest moving it to the assembly file, in the "Preface", so that it structurally applies to the entire document.